config/manager/manager.yaml

```diff
@@ -23,9 +23,7 @@ spec:
       securityContext:
         runAsNonRoot: true
       containers:
-        - command:
```

Comment:
Why have we removed the command block?

Reply:
It's not needed.

`config/manager/manager.yaml`
-----------------------------

Remove `command` from `containers` as we have `ENTRYPOINT ["/manager"]` in `build/manager/Dockerfile`, i.e., the block
below is not needed:

```yaml
        - command:
            - /manager
```